include/irr/asset/IMeshPacker.h

```diff
@@ -107,9 +129,6 @@ class IMeshPacker
         }
     }
 
-
-    virtual void commit(/*TODO*/) = 0;
```

Comment:
retain `alloc`, `commit` and both planned `free` overrides in the base class
https://github.com/buildaworldnet/IrrlichtBAW/pull/496#discussion_r443521117
```cpp
template<typename Iterator>
ReservedAllocationMeshBuffers alloc(Iterator _begin, Iterator _end)
{
    // do some logic common to all MeshBufferTypes
    ...    
    // do some logic specific
   for (auto it=_begin; it!=_end; it++)
       alloc_impl(*it); // now `alloc_impl` can be virtual and non templated!
}

template<typename Iterator>
PackedMeshBuffers commit(ReservedAllocationMeshBuffers& _reservation, Iterator _begin, Iterator _end)
{
    // do some logic common to all MeshBufferTypes
    ...    
    // do some logic specific
   uint32_t i = 0u;
   for (auto it=_begin; it!=_end; it++)
       commit_impl(_reservation,i,*it); // now `commit_impl` can be virtual and non templated!
}

virtual free(PackedMeshBuffers& _commitedData);

virtual free(ReservedAllocationMeshBuffers& _reservation);
```